.circleci/config.yml

```diff
@@ -12,6 +12,9 @@ jobs:
       # CircleCI maintains a library of pre-built images
       # documented at https://circleci.com/docs/2.0/circleci-images/
       # - image: circleci/mongo:3.4.4
+      
+    branches:
+      ignore: gh-pages
```

Comment:
This is ignored b/c it is automatically generated? Should we have a test in place to test that there were no markdown errors or otherwise when generating?

Reply:
Yes, tests for the website should still run from the master branch (we don't have any yet!). But the `gh-pages` is auto-generated from Docusaurus, so there's no need to run tests on it.